codacy/values.yaml

```diff
@@ -128,49 +130,56 @@ global:
 
 portal:
   replicaCount: 1
+  fullnameOverride: codacy-portal
 
 activities:
   replicaCount: 1
-
+  fullnameOverride: codacy-activities
   activitiesdb:
     create: true
-    host: codacy-activitiesdb.codacy.svc.cluster.local
+    host: codacy-activitiesdb
     postgresqlUsername: "codacy"
     postgresqlDatabase: "activities"
     postgresqlPassword: "PLEASE_CHANGE_ME"
     service:
       port: 5432
+    fullnameOverride: codacy-activitiesdb
 
 remote-provider-service:
   replicaCount: 1
+  fullnameOverride: codacy-remote-provider-service
 
 hotspots-api:
   replicaCount: 1
-
+  fullnameOverride: codacy-hotspots-api
   hotspotsdb:
     create: true
-    host: codacy-hotspotsdb.codacy.svc.cluster.local
+    host: codacy-hotspotsdb
     postgresqlUsername: codacy
     postgresqlDatabase: hotspots
     postgresqlPassword: codacy
     service:
       port: 5432
+    fullnameOverride: codacy-hotspotsdb
 
 hotspots-worker:
   replicaCount: 1
+  fullnameOverride: codacy-hotspots-worker
 
 listener:
   replicaCount: 1
+  fullnameOverride: codacy-listener
   config:
     play:
       configFile: application.enterprise.conf
       logFile: logback.xml
-
+  
   persistence:
     claim:
       size: 100Gi
 
   nfsserverprovisioner:
+    fullnameOverride: codacy-nfsserverprovisioner
```

Comment:
I don't think this one is necessary. We never call this. 
If we do, we need to find out where to make sure its being used like this.

Reply:
@DReigada do we use this?

Reply:
This should worker as expected
Although it is not documented in [here](https://github.com/helm/charts/blob/master/stable/nfs-server-provisioner/values.yaml)
It is used by the templates in the [helm chart](https://github.com/helm/charts/blob/master/stable/nfs-server-provisioner/templates/_helpers.tpl#L16)